src/tools/common.ts

```diff
@@ -1,7 +1,8 @@
 import { KeyInfo } from '../fetchers/BitIndexer.type'
 import { ChainType, CoinType, EvmChainId, EvmCoinTypes } from '../const'
-import { formatsByName } from '@ensdomains/address-encoder'
+import { utils } from 'ethers'
```

Comment:
`utils` and `tronweb` are too large; can we make them lighter? Like `import {isAddress} from 'ethers/utils'`; `import {isAddress} from 'tronweb'`